From b7be14e77f166f909c61ed57353eafd4e01bf85f Mon Sep 17 00:00:00 2001
From: Nikolay Kulikov <nikolayof23@gmail.com>
Date: Fri, 17 Jul 2026 21:52:02 +0300
Subject: [PATCH 07585/15333] staging: rtl8723bs: remove unused
 'bIQKInitialized ' from hal_com_data

This field is always initialized to 'false' but is never used.

Signed-off-by: Nikolay Kulikov <nikolayof23@gmail.com>
Link: https://patch.msgid.link/20260717185407.56513-4-nikolayof23@gmail.com
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723bs/hal/rtl8723b_hal_init.c | 3 ---
 drivers/staging/rtl8723bs/include/hal_data.h      | 1 -
 2 files changed, 4 deletions(-)

diff --git a/drivers/staging/rtl8723bs/hal/rtl8723b_hal_init.c b/drivers/staging/rtl8723bs/hal/rtl8723b_hal_init.c
index 5be6fe38d451..f3f9b07fd1d4 100644
--- a/drivers/staging/rtl8723bs/hal/rtl8723b_hal_init.c
+++ b/drivers/staging/rtl8723bs/hal/rtl8723b_hal_init.c
@@ -1048,12 +1048,9 @@ void rtl8723b_init_default_value(struct adapter *padapter)
 
 	/*  init default value */
 	pHalData->fw_ractrl = false;
-	pHalData->bIQKInitialized = false;
 	if (!adapter_to_pwrctl(padapter)->bkeepfwalive)
 		pHalData->LastHMEBoxNum = 0;
 
-	pHalData->bIQKInitialized = false;
-
 	/*  init dm default value */
 	pdmpriv->TM_Trigger = 0;/* for IQK */
 /* 	pdmpriv->binitialized = false; */
diff --git a/drivers/staging/rtl8723bs/include/hal_data.h b/drivers/staging/rtl8723bs/include/hal_data.h
index 8b1b61e78fee..532b1635831d 100644
--- a/drivers/staging/rtl8723bs/include/hal_data.h
+++ b/drivers/staging/rtl8723bs/include/hal_data.h
@@ -206,7 +206,6 @@ struct hal_com_data {
 	u8 TypeAPA;
 	u8 RFEType;
 	u8 BoardType;
-	u8 bIQKInitialized;
 	bool		bLCKInProgress;
 
 	bool		bSwChnl;
-- 
2.55.0

